builtin/logical/nomad/path_config_lease.go

```diff
@@ -13,6 +13,12 @@ const leaseConfigKey = "config/lease"
 func pathConfigLease(b *backend) *framework.Path {
 	return &framework.Path{
 		Pattern: "config/lease",
+
+		DisplayAttrs: &framework.DisplayAttributes{
+			OperationPrefix: operationPrefixNomad,
+			OperationSuffix: "lease-config",
```

Comment:
same here, what about 
```suggestion
			OperationSuffix: "config-lease",
```

Reply:
same as above

Reply:
Updated to:
```
nomad-read-lease-config
nomad-configure-lease
nomad-delete-lease-config
```